tests/test_images.py

```diff
@@ -1,21 +1,25 @@
 import pytest
+import uuid
 from io import BytesIO
 from aiodocker import utils
 from aiodocker.exceptions import DockerError
 
 
+def _random_name():
+    return "aiodocker-" + uuid.uuid4().hex[:7]
+
+
 @pytest.mark.asyncio
 async def test_build_from_remote_file(docker):
     remote = ("https://raw.githubusercontent.com/aio-libs/"
               "aiodocker/master/tests/docker/Dockerfile")
 
-    tag = "image:1.0"
+    tag = "%s:1.0" % _random_name()
```

Comment:
As I said it just a matter of consistency :) remember that is `.format()` and not `f""`, that is available only from python 3.6.